server/testfixtures/test-repos/workspace-parallel-yaml/exp-output-apply-all-staging.txt

```diff
@@ -1,3 +1,33 @@
+Ran Apply for 2 projects:
```

Comment:
this file contains staging in the name meaning that the output should only contain staging in it. Something is wrong here since it's attempting to apply both production and staging.  The fixtures were correct as is.